File: QuickStart/PageView/PageViewImages.cs
using System;
using System.Collections.Generic;
using System.Text;
using Telerik.WinControls;
using System.Drawing;

namespace Telerik.Examples.WinControls.PageView
{
    class PageViewImages
    {
        public static Image[] Images;
        public static string[] Names;

        static PageViewImages()
        {
            Names = new string[] { 
                "Brush.png", "Calendar 1.png", "CD Burner.png", 
                "Chart Pie.png", "Configuration Tools.png", "Configuration.png", 
                "Dice.png", "Download.png", "Home.png", "Logout.png" };

            Type type = typeof(PageViewImages);
            string path = "Telerik.Examples.WinControls.PageView.Resources.";
            Images = new Image[10];

            for (int i = 0; i < 10; i++)
            {
                Images[i] = ResourceHelper.ImageFromResource(type, path + Names[i]);
            }
        }
    }
}

File: QuickStart/PageView/PageViewImages.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports Telerik.WinControls
Imports System.Drawing

Namespace Telerik.Examples.WinControls.PageView
	Friend Class PageViewImages
		Public Shared Images() As Image
		Public Shared Names() As String

		Shared Sub New()
			Names = New String() { "Brush.png", "Calendar 1.png", "CD Burner.png", "Chart Pie.png", "Configuration Tools.png", "Configuration.png", "Dice.png", "Download.png", "Home.png", "Logout.png" }

			Dim type As Type = GetType(PageViewImages)
			Images = New Image(9){}

			For i As Integer = 0 To 9
				Images(i) = ResourceHelper.ImageFromResource(type, Names(i))
			Next i
		End Sub
	End Class
End Namespace
